lib/src/main/java/com/diffplug/spotless/npm/NpmFormatterStepStateBase.java

```diff
@@ -58,7 +62,7 @@ protected NpmFormatterStepStateBase(String stepName, Provisioner provisioner, Np
 		this.jarState = JarState.from(j2v8MavenCoordinate(), requireNonNull(provisioner));
 
 		this.nodeModulesDir = prepareNodeModules(buildDir, npm);
-		this.nodeModulesSignature = FileSignature.signAsList(this.nodeModulesDir);
+		this.packageJsonSignature = FileSignature.signAsList(packageJsonFile(nodeModulesDir));
```

Comment:
I refactored the code slightly to avoid using string file names in multiple places. `NodeServerLayout` class exposes `node-modules`, `package.json` and `serve.js` via API methods. This removes the need for a comment. Please let me know if you think that's too much :)